packages/itmat-docker/launcher.js

```diff
@@ -20,7 +20,10 @@ Promise.all([
 
     routers.forEach((router) => {
         // Linking itmat's router on /api
-        root.use('/', router);
+        if (router.hasOwnProperty('proxies'))
+            root.use('/', router.getApp());
+        else
+            root.use('/', router);

```

Comment:
Maybe rather than do this it would be more elegant to change the `job-executor` to have the same new shape as the `interface` ?